service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DraftCMOControllerTest.java

```diff
@@ -56,8 +58,11 @@
     private ObjectMapper mapper;
 
     @Test
-    void aboutToStartCallbackShouldPopulateHearingDatesList() throws Exception {
-        Map<String, Object> data = ImmutableMap.of("hearingDetails", hearingDetails);
+    void aboutToStartCallbackShouldPopulateHearingDatesListAndSetAssigneeDropdownKeys() throws Exception {
```

Comment:
Does this test do too much? Would it make more sense to split into two tests? Keep what we had before (populate hearing dates list) and then another test for setting assignee dropdown keys, they seem to be unrelated

Reply:
No because we would be doing the same setup in both tests just separating out the assertions. It's better to have one the int test with multiple assertions.